### DISCUSSION TITLE
Can we script or run programatically (not using the UI)?

### DISCUSSION THREAD

**Discussion by KyleCBowman** [lllyasviel/Fooocus#2483] [Q&A] [answered]
Firstly, the UI is brilliant and works really well.

However, I was looking to see if we can script it to run different prompts one after another.

For example, I am looking to build several designs for my logo and I would like to set it to try the same prompt but with "red", or "blue" appended to the end. At the moment, the only way I can do this in the UI is to run it, wait and then edit the prompt, run it again.

Is this possible?

**Comment by mashb1t** [comment, marked as answer]
You can do this in three ways:
1. one prompt after another
1. using the new array syntax with image number 2: [[red,blue]]
1. using the Websockets API

I'd suggest to use option number 2.

**Comment by KyleCBowman** [comment]
Hey thank you very much, thats a great feature, when did it get added?

**Reply by mashb1t** [reply]
version 2.2.0, https://github.com/lllyasviel/Fooocus/releases/tag/2.2.0

**Reply by KyleCBowman** [reply]
Awesome thanks!

**Comment by KyleCBowman** [comment]
using the new array syntax with image number 2: [[red,blue]]
